go/memcached-operator/Makefile

```diff
@@ -4,18 +4,18 @@ NAMESPACE=memcached
 
 ##@ Application
 
-install: ## Install all resources (CR/CRD's, RBCA and Operator)
+install: ## Install all resources (CR/CRD's, RBAC and Operator)
 	@echo ....... Creating namespace ....... 
 	- kubectl create namespace ${NAMESPACE}
-	@echo ....... Applying CRDS and Operator .......
+	@echo ....... Applying CRDs .......
 	- kubectl apply -f deploy/crds/cache.example.com_memcacheds_crd.yaml -n ${NAMESPACE}
 	@echo ....... Applying Rules and Service Account .......
 	- kubectl apply -f deploy/role.yaml -n ${NAMESPACE}
 	- kubectl apply -f deploy/role_binding.yaml  -n ${NAMESPACE}
 	- kubectl apply -f deploy/service_account.yaml  -n ${NAMESPACE}
 	@echo ....... Applying Operator .......
 	- kubectl apply -f deploy/operator.yaml -n ${NAMESPACE}
-	@echo ....... Creating the Database .......
+	@echo ....... Creating the CRs .......
 	- kubectl apply -f deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml -n ${NAMESPACE}
```

Comment:
Thank you for your contribution. 🥇 
Could you update the uninstall a well?

Reply:
The uninstall part is correct. could you point me where need to update? Thanks.

Reply:
Sorry, it is true. I assume that the text was equally wrong :-)